engine/gamesys/src/gamesys/components/comp_gui.cpp

```diff
@@ -1485,6 +1485,46 @@ namespace dmGameSystem
         }
     }
 
+    static uint32_t CalcVertexCount(dmGui::HScene scene, GuiWorld* gui_world, dmGraphics::HTexture texture, const dmGui::RenderEntry* entries, uint32_t node_count)
```

Comment:
Just like for sprites, we need to calculate the actual number of vertices needed, before we generate the vertices.